docs/maintainers/issue-management.md

```diff
@@ -8,6 +8,7 @@ Issues are opened for many different reasons.
 We use the following labels for issue classifications:
 
 * `Issue-Bug`: the issue is reporting a bug
+* `Issue-Code Cleanup`: the issue is for cleaning up the code with no impact on functionality
```

Comment:
I originally had no space, but looking at some of the other labels we have, I decided to have the space for consistency.

`Issue-Code Cleanup` is to identify known issues in the code where we should clean up (I've already labeled a number of such issues) that isn't a priority to address so they aren't automatically PRs